.gitignore

```diff
@@ -38,6 +38,13 @@ makefile*
 *.sln
 *.vcxproj
 
+# Cake projects
+tools/*dll
```

Comment:
[Cake recommends](https://cakebuild.net/docs/tutorials/pinning-cake-version#gitignore) the following exclusion patterns:
```
tools/*
!tools/packages.config
```